apps/client/package.json

```diff
@@ -21,11 +21,13 @@
     "@emotion/styled": "11.11.0",
     "@mui/icons-material": "5.11.16",
     "@mui/material": "5.13.5",
+    "@types/react-router-dom": "^5.3.3",
     "@vitejs/plugin-react-swc": "3.3.2",
+    "axios": "^1.4.0",
     "mapbox-gl": "2.15.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-router-dom": "6.14.0",
+    "react-router-dom": "^6.14.0",
```

Comment:
Don't change the dependencies already defined, remove the upper sans of this line